week0002/phucnm/C++/week002.cpp

```diff
@@ -0,0 +1,56 @@
+#include <cstdio>
+#include <cstring>
+
+void spiralPrinting(int nRow, int nCol) {
```

Comment:
When speaking about size of arrays of containers, we should use `size_t` instead of `int`. It's already included in `<cstring>`:

http://en.cppreference.com/w/cpp/types/size_t